modules/perception/lidar/lib/detection/lidar_point_pillars/point_pillars_detection.cc

```diff
@@ -13,15 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *****************************************************************************/
-#include <cuda_runtime_api.h>
+#include "modules/perception/lidar/lib/detection/lidar_point_pillars/point_pillars_detection.h"
+
+#include <algorithm>
+#include <cstring>
+#include <numeric>
+#include <random>
 #include <vector>
 
 #include "cyber/common/log.h"
+#include "include/cuda_runtime_api.h"
```

Comment:
Generally, we just `#include "cuda_runtime_api.h"`.

Reply:
Use `#include <cuda_runtime_api.h>` instead.